### PR TITLE
installed and used kvpair

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: src/utils/containerCode.ts [hunk 1]
```diff
@@ -28,8 +27,11 @@ const createRequestHandler: RuntimeRequestHandler = async (
  * By including the createRequestHandler, we can create any droplet types we include in the registry on-demand.
  * These can then be retrieved via container.request("/dataObjectId").
  */
-export const KeyValueContainerRuntimeFactory = new BaseContainerRuntimeFactory(
-    [KeyValueInstantiationFactory.registryEntry],
-    [],
-    [createRequestHandler, innerRequestHandler]
-);
+export const getRuntimeFactory = (factories: any[]) => {
```
Comment (ChumpChief) [inline src/utils/containerCode.ts:30]: Maybe DataObjectFactory rather than any?

Comment (micahgodbolt) [reply src/utils/containerCode.ts:30]: Yeah, wasn't sure which interface..i'll try that

Comment (micahgodbolt) [reply src/utils/containerCode.ts:30]: image

Yeah, i had tried that. DataObjectFactory has a bunch of generics

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
         "start:server": "tinylicious"
     },
     "dependencies": {
+        "@fluid-experimental/data-objects": "^0.34.0-14942",
```
Comment (ChumpChief) [inline package.json:16]: Prob want to be consistent on version used, recommend not using 0.34 until release is finalized

Comment (micahgodbolt) [reply package.json:16]: Yeah, 0.34 is coming out tomorrow. Just wanted to see if I could get this working. happy to wait on merge until 0.34 is out

File: src/utils/containerCode.ts (hunk 1 above)
Comment (ChumpChief) [reply src/utils/containerCode.ts:30]: Oof, ok - maybe something to work on later then.

File: package.json (hunk 2 above)
Comment (ChumpChief) [reply package.json:16]: At that time probably also update the other dependencies to 0.34 too then.

**Review by ChumpChief — APPROVED** [review]

**Pull request merged**